### DISCUSSION TITLE
How to create a shortcut for frog ocr to immediately go into select area mode?

### DISCUSSION THREAD

**Discussion by shahiljain** [TenderOwl/Frog#125] [Q&A]
Basically I want to set a keyboard shortcut to start frog's select an area to screenshot mode. I am wondering if there is a commandline command. If so then I can use gnome's keyboard settings to create a quick keyboard shortcut. Thanks!

**Comment by amka** [comment]
Yeah, there is a way to do that:
```
flatpak run com.github.tenderowl.frog -e
```

It's based on #17. I don't use it often so tell me if it goes wrong.

**Comment by heyvoon** [comment]
Just use https://github.com/dynobo/normcap
It seems like this project is pretty dead unfortunately.
